src/main.py

```diff
@@ -0,0 +1,38 @@
+def main():
+    add_fifteen = lambda x: x + 15
+    print(add_fifteen(10))
+    product = lambda x, y: x / y
+    print(product(10, 2))
+    to_sort = [('English', 88), ('Science', 90), ('Maths', 97), ('Social sciences', 82)]
+    to_sort.sort(key=lambda x: x[1])
+    print(to_sort)
+    my_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    squares = list(map(lambda x: x ** 2, my_list))
+    print(squares)
+    cubes = list(map(lambda x: x ** 3, my_list))
+    print(cubes)
+    lines = ["10000101011", "111111", "01010101010010", "011001100001", "001010101011"]
+    filtered = len(list(filter(lambda x: not "11" in x, lines)))
```

Comment:
a z countem masz pomysl jak to zrobić? count wtedy byłby w lambda

Reply:
(nonblocking; nitpicking)

Reply:
filtered_lines = list(filter(lambda x: x.count("11") < 1, lines))